test/test_thread_safe_queue.cc

```diff
@@ -44,53 +45,53 @@ TEST(ThreadSafeQueue, Simple) {
   atomic<bool> consumer_do_wait{true};
   atomic<int> item_count{0};
 
-  ngraph::Event::enable_event_tracing();
+  ngraph::runtime::event::Manager::enable_event_tracing();
```

Comment:
This may not be required. Is probably here as a stray comment. 
We can remove this if the test is not using the trace.